packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -453,7 +459,7 @@ class DatePicker extends React.Component {
             onBlur={this.handleBlur}
             onChange={this.handleChange}
             onChangeRaw={this.handleChangeRaw}
-            onClickOutside={this.handleOnClickOutside}
+            onDismiss={onDismiss || this.handleOnClickOutside}
```

Comment:
Should we keep `onClickOutside={this.handleOnClickOutside}` and only do `onDismiss={onDismiss}` for this [reason](https://github.com/cerner/terra-framework/pull/1224#discussion_r476666155)?

Reply:
Reverted here https://github.com/cerner/terra-framework/pull/1224/commits/9d8be2a715656afc67046abb501756e8ef610399